.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+  schedule:
+    - cron: '0 1 * * *' # Test every day at 1AM
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Add conda to system path
+        run: |
+          echo $CONDA/bin >> $GITHUB_PATH 
+
+      - name: Install dependencies
+        run: |
+          conda env update --file environment.yml --name base
+
+      - name: Conda list
+        run: |
+          conda list
+
+      - name: Install topcoffea package
+        run: |
+          pip install -e .
+
+      - name: Download root files
+        run: |
+          wget http://www.crc.nd.edu/~ywan2/root_files/NAOD-00000_18449.root
+
+      - name: Run processors over test files
```

Comment:
Maybe change this to "Run processors over test files with futures executor" to prepare for adding the WQ executor, which could be done in a later PR.